reader/nordigen/nordigen.go

```diff
@@ -40,12 +40,37 @@ func transactionsToYnabber(account ynabber.Account, t nordigen.AccountTransactio
 			return nil, fmt.Errorf("failed to parse string to time: %w", err)
 		}
 
+		// Find payee
+		payee := ynabber.Payee("")
+		for _, source := range config.YNAB.PayeeSource {
+			if payee == "" {
+				if source == "name" {
+					// Creditor/debtor name can be used as is
+					if v.CreditorName != "" {
+						payee = ynabber.Payee(v.CreditorName)
+					} else if v.DebtorName != "" {
+						payee = ynabber.Payee(v.DebtorName)
+					}
+				} else if source == "unstructured" {
+					// Unstructured data may need some formatting
+					payee = ynabber.Payee(v.RemittanceInformationUnstructured)
+					s, err := payee.Parsed(config.YNAB.PayeeStrip)
+					if err != nil {
+						log.Printf("Failed to parse payee: %s: %s", string(payee), err)
+					}
+					payee = ynabber.Payee(s)
+				} else {
+					log.Panicf("Unrecognized payee data source %s", source)
```

Comment:
We should not panic here, instead return nil and an error.

Reply:
```suggestion
					return nil, fmt.Errorf("Unrecognized payee data source %s", source)
```

Reply:
I thought we should exit completely. If payee suorce definition is wrong it is not going to work, and running in loop will not help. User must fix the environment. Or does this return also cause Ynabber to exit?

Although even better option would be to check everything already when reading the configuration or do both.

Reply:
If you return an error and it's not handled by the caller it will eventually reach the panic in the main() function.